lib/instrumental/agent.rb

```diff
@@ -42,18 +44,28 @@ def initialize(api_key, options = {})
 
       # defaults
       # host:        collector.instrumentalapp.com
-      # port:        8000
+      # port:        8001
       # enabled:     true
       # synchronous: false
+      # secure:      true
+      # verify:      true
       @api_key         = api_key
       @host, @port     = options[:collector].to_s.split(':')
       @host          ||= 'collector.instrumentalapp.com'
-      @port            = (@port || 8000).to_i
+      desired_secure   = options[:secure].nil? ? allows_secure? : !!options[:secure]
+      if !allows_secure? && desired_secure
+        @logger.warn "Cannot connect to Instrumental via encrypted transport, SSL not available"
+        desired_secure = false
```

Comment:
I think we could, but it might require creating a new signaling mechanism for the agent. ( Otherwise we end up in a weird situation where we've acknowledged we don't have a secure connection and can't create one, but we do create an insecure connection to create and send only one flag metric )